### PR TITLE
RHIDP-2596: update OIDC auth provider configurations

### PR DESCRIPTION
**Pull request by Zaperex**
### Description
Update the OIDC Auth Provider documentation to reflect the changes made in https://github.com/janus-idp/backstage-showcase/pull/1309

### CODE REVIEW

File: modules/getting-started/con-oidc-overview.adoc [hunk 1]
```diff
@@ -22,5 +22,16 @@ auth:
         # tokenEndpointAuthMethod: ${AUTH_OIDC_TOKEN_ENDPOINT_METHOD}
         # tokenSignedResponseAlg: ${AUTH_OIDC_SIGNED_RESPONSE_ALG}
         # scope: ${AUTH_OIDC_SCOPE}
+        ## Auth provider will try each sign-in resolver until it succeeds. Comment out the resolvers you do not want
```
Comment (hmanwani-rh) [inline modules/getting-started/con-oidc-overview.adoc:25]:
```suggestion
        ## The auth provider will try each sign-in resolver until it succeeds. Comment out the resolvers you do not want to use.
```

File: modules/getting-started/con-oidc-overview.adoc [hunk 2]
```diff
@@ -22,5 +22,16 @@ auth:
...
+            # - resolver: emailMatchingUserEntityProfileEmail
+            # - resolver: emailLocalPartMatchingUserEntityName
 signInPage: oidc
-----
\ No newline at end of file
+----
+
+[NOTE]
+====
+If no resolvers are provided, the OIDC authentication provider will default to no resolvers and will not function.
```
Comment (hmanwani-rh) [inline modules/getting-started/con-oidc-overview.adoc:36]:
```suggestion
If no resolvers are provided, the OIDC authentication provider defaults to having no resolvers and therefore does not function.
```

**Review by hmanwani-rh — CHANGES_REQUESTED** [review]
@Zaperex Thank you for creating the PR and incorporating these changes into the docs. I've added a couple of minor suggestions. Could you please review them and let me know if you have any questions?

**Review by zdrapela — APPROVED** [review]
Looking good to me! 👍️

**Review by hmanwani-rh — APPROVED** [review]

**Pull request merged**